similar/main/inferno.cpp

```diff
@@ -95,7 +95,7 @@ char copyright[] = "DESCENT II  COPYRIGHT (C) 1994-1996 PARALLAX SOFTWARE CORPOR
 #endif
 #include "event.h"
 #include "rbaudio.h"
-#ifndef __linux__
+#if !defined (__linux__) || !defined (__OpenBSD__)
```

Comment:
Both of these hunks have the same concern as in `med.cpp`.

Reply:
same as above - should be &&, not ||